ledger-core/v2/application/testwalletapi/handlers.go

```diff
@@ -175,7 +175,43 @@ func (s *TestWalletServer) GetBalance(w http.ResponseWriter, req *http.Request)
 		TraceID: traceID,
 		Error:   "",
 	}
+
 	defer func() { s.mustWriteResult(w, result) }()
+
+	ref, err := insolar.NewReferenceFromString(params.WalletRef)
+
+	if err != nil {
+		result.Error = fmt.Sprintf("Failed to create reference from string (%s)", params.WalletRef)
+		return
+	}
+
+	walletReq := payload.VCallRequest{
+		CallType:       payload.CTMethod,
+		Callee:         *ref,
+		CallSiteMethod: getBalance,
+	}
+
+	walletRes, err := s.runWalletRequest(ctx, walletReq)
+
+	if err != nil {
+		result.Error = err.Error()
+		return
+	}
+
+	var (
+		amount          uint32
+		contractCallErr *foundation.Error
+	)
+
+	err = foundation.UnmarshalMethodResultSimplified(walletRes.ReturnArguments, &amount, &contractCallErr)
+	switch {
+	case err != nil:
+		result.Error = errors.Wrap(err, "Failed to unmarshal response").Error()
```

Comment:
You wrap error here and didn't do that on line 197. Why?

Reply:
fixed